ConcurrentPercolator/ConcurrentPercolator.tla

```diff
@@ -414,7 +416,7 @@ AbortedConsistency ==
     (/\ client_state[c] = "aborted"
      /\ client_ts[c].commit_ts # 0
```

Comment:
Looks like this premise (`client_ts[c].commit_ts # 0`) is no longer an valid one.

Reply:
Why? In the `init` status, the commit_ts is 0.